supporting-blog-content/customer-success-example/README.md

```diff
@@ -0,0 +1,35 @@
+Example of AI Conversational Search using Elasticsearch, OpenAI and Vercel. 
+
+# Load the data
+
+Export the following environment variables: 
+- ELASTICSEARCH_URL: You can find it in your Elasticsearch deployment
+- ES_API_KEY: Your Elasticsearch API Key
+
+Run the script ingest.py: `python ingest.py` to index the data. 
+
+
+# Chat application
+
+## Install dependencies
+
+Go to the folder `chat-example/app` and execute `yarn install` to install the project dependencies. 
+
+## Provide environment variable
+
+Export the following environment variables: 
+- ELASTICSEARCH_URL: You can find it in your Elasticsearch deployment
+- ES_API_KEY: Your Elasticsearch API Key
+- OPENAI_API_KEY: Your OpenAI API Key
```

Comment:
Can you add a section on prerequisites above. It's useful to let people know where to go to create the API key, and also make clear they need to create an Elasticsearch deployment. We can't assume prerequisites are obvious.